app/views/projects/show.html.erb

```diff
@@ -269,175 +180,289 @@
01C9.1826 2.10032 9.09465 1.90498 8.9381 1.76095C8.78155 1.61692 8.56922 1.53601 8.34782 1.53601C8.12642 1.53601 7.91409 1.61692 7.75754 1.76095C7.60098 1.90498 7.51303 2.10032 7.51303 2.30401V3.07201C7.51303 3.2757 7.60098 3.47104 7.75754 3.61507C7.91409 3.7591 8.12642 3.84001 8.34782 3.84001C8.56922 3.84001 8.78155 3.7591 8.9381 3.61507C9.09465 3.47104 9.1826 3.2757 9.1826 3.07201V2.30401ZM13.0702 4.42139C13.2222 4.27654 13.3064 4.08254 13.3045 3.88117C13.3026 3.67981 13.2148 3.48718 13.06 3.34479C12.9052 3.2024 12.6959 3.12163 12.477 3.11988C12.2581 3.11813 12.0472 3.19554 11.8898 3.33543L11.2996 3.87841C11.1475 4.02326 11.0634 4.21726 11.0653 4.41862C11.0672 4.61999 11.155 4.81261 11.3098 4.95501C11.4646 5.0974 11.6739 5.17817 11.8928 5.17992C12.1117 5.18167 12.3225 5.10426 12.48 4.96436L13.0702 4.42139ZM15.0261 7.68001C15.0261 7.8837 14.9381 8.07904 14.7816 8.22307C14.625 8.3671 14.4127 8.44801 14.1913 8.44801H13.3565C13.1351 8.44801 12.9228 8.3671 12.7662 8.22307C12.6097 8.07904 12.5217 7.8837 12.5217 7.68001C12.5217 7.47632 12.6097 7.28098 12.7662 7.13695C12.9228 6.99292 13.1351 6.91201 13.3565 6.91201H14.1913C14.4127 6.91201 14.625 6.99292 14.7816 7.13695C14.9381 7.28098 15.0261 7.47632 15.0261 7.68001ZM4.21564 4.96436C4.29265 5.03771 4.38476 5.09622 4.48661 5.13647C4.58846 5.17672 4.698 5.19791 4.80884 5.19879C4.91968 5.19968 5.0296 5.18025 5.1322 5.14163C5.23479 5.10302 5.32799 5.04599 5.40637 4.97388C5.48475 4.90177 5.54674 4.81602 5.58871 4.72164C5.63069 4.62725 5.65181 4.52612 5.65084 4.42415C5.64988 4.32218 5.62685 4.2214 5.5831 4.1277C5.53935 4.034 5.47576 3.94926 5.39603 3.87841L4.80583 3.33543C4.64839 3.19554 4.43752 3.11813 4.21865 3.11988C3.99977 3.12163 3.7904 3.2024 3.63562 3.34479C3.48084 3.48718 3.39305 3.67981 3.39115 3.88117C3.38925 4.08254 3.47339 4.27654 3.62545 4.42139L4.21564 4.96436ZM4.1739 7.68001C4.1739 7.8837 4.08595 8.07904 3.9294 8.22307C3.77285 8.3671 3.56052 8.44801 3.33912 8.44801H2.50434C2.28294 8.44801 2.07061 8.3671 1.91406 8.22307C1.75751 8.07904 1.66956 7.8837 1.66956 7.68001C1.66956 7.47632 1.75751 7.28098 1.91406 7.13695C2.07061 6.99292 2.28294 6.91201 2.50434 6.91201H3.33912C3.56052 6.91201 3.77285 6.99292 3.9294 7.13695C4.08595 7.28098 4.1739 7.47632 4.1739 7.68001ZM6.67825 12.288V11.52H10.0174V12.288C10.0174 12.6954 9.84148 13.0861 9.52838 13.3741C9.21527 13.6622 8.79061 13.824 8.34782 13.824C7.90502 13.824 7.48036 13.6622 7.16726 13.3741C6.85415 13.0861 6.67825 12.6954 6.67825 12.288ZM10.0174 10.752C10.0299 10.4909 10.191 10.2559 10.4156 10.0923C10.9604 9.69698 11.3579 9.15531 11.5527 8.54259C11.7475 7.92988 11.7299 7.27655 11.5025 6.67341C11.2751 6.07028 10.8491 5.54731 10.2838 5.17719C9.71838 4.80707 9.04172 4.60818 8.34782 4.60818C7.65392 4.60818 6.97725 4.80707 6.41188 5.17719C5.8465 5.54731 5.4205 6.07028 5.19309 6.67341C4.96569 7.27655 4.94817 7.92988 5.14297 8.54259C5.33777 9.15531 5.73521 9.69698 6.28006 10.0923C6.50545 10.2559 6.66573 10.4909 6.67742 10.752H10.0182H10.0174Z" fill="white"/>
```

Comment:
We load all `svg`s with `inline_svg_pack_tag`.

https://github.com/helpwithcovid/covid-volunteers/blob/d2ddd086dd931d46f9609cc32afbadb2a6b1a7d4/app/views/home/index.html.erb#L11